### PR TITLE
Fixes #14384: Update mvn direct call in rudder-webapp package makefile

### PR DESCRIPTION
**Pull request by fanf**
https://issues.rudder.io/issues/14384

### CODE REVIEW

File: rudder-webapp/SOURCES/Makefile [hunk 1]
```diff
@@ -112,18 +112,15 @@ maven/bin/mvn:
...
 ifeq (false,$(NRM))
-	if [ ! -f endpoint.war ]; then cd rudder-sources/rudder-parent-pom && ../../maven/bin/mvn -s settings-external.xml --batch-mode -Dmaven.repo.local=.m2/repository -Dmaven.test.skip=true -U install; fi
-	if [ ! -f endpoint.war ]; then cd rudder-sources/rudder-commons    && ../../maven/bin/mvn -s settings-external.xml --batch-mode -Dmaven.repo.local=.m2/repository -Dmaven.test.skip=true -U install; fi
-	if [ ! -f endpoint.war ]; then cd rudder-sources/scala-ldap        && ../../maven/bin/mvn -s settings-external.xml --batch-mode -Dmaven.repo.local=.m2/repository -Dmaven.test.skip=true -U install; fi
-	if [ ! -f endpoint.war ]; then cd rudder-sources/ldap-inventory    && ../../maven/bin/mvn -s settings-external.xml --batch-mode -Dmaven.repo.local=.m2/repository -Dmaven.test.skip=true -U install; fi
-	if [ ! -f endpoint.war ]; then mv rudder-sources/rudder/rudder-web/target/inventory-provisioning-web*.war endpoint.war; fi
+	if [ ! -f endpoint.war ]; then cd rudder-sources/ && ../../maven/bin/mvn -s settings-external.xml --batch-mode -Dmaven.repo.local=.m2/repository -Dmaven.test.skip=true -U install; fi
```
Comment (peckpeck) [inline rudder-webapp/SOURCES/Makefile]: In sources there one directory per repo, so this should be at least 
cd rudder-sources/rudder

File: rudder-webapp/SOURCES/Makefile [hunk 2]
```diff
@@ -112,18 +112,15 @@ maven/bin/mvn:
 rudder.war: maven/bin/mvn rudder-sources endpoint.war
 	maven/bin/mvn -s settings-external.xml --batch-mode org.apache.maven.plugins:maven-dependency-plugin:3.0.0:copy -Dmaven.repo.local=.m2/repository -Dartifact=com.normation.rudder:rudder-web:$$(sed -n '/<parent>/,/<.parent>/s/.*<version>\(.*\)<.version>.*/\1/p' rudder-sources/rudder/pom.xml):war -DoutputDirectory=. && mv rudder-web*.war rudder.war $(NEXUS_IGNORE)
 ifeq (false,$(NRM))
-	if [ ! -f rudder.war ]; then cd rudder-sources/rudder            && ../../maven/bin/mvn -s settings-external.xml --batch-mode -Dmaven.repo.local=.m2/repository -Dmaven.test.skip=true -U install; fi
+	if [ ! -f rudder.war ]; then cd rudder-sources/ && ../../maven/bin/mvn -s settings-external.xml --batch-mode -Dmaven.repo.local=.m2/repository -Dmaven.test.skip=true -U install; fi
```
Comment (peckpeck) [inline rudder-webapp/SOURCES/Makefile]: I think rdder should stay here

File: rudder-webapp/SOURCES/Makefile (hunk 1 above)
Comment (fanf) [reply rudder-webapp/SOURCES/Makefile]: ok, perhaps I misunderstood what is in source. In git repository, I want to be at the path: `rudder-git/sources/webapp/[HERE]`.
What should be the path? `cd rudder-sources/rudder/sources/webapp` ?

**Comment by fanf** [comment]
@amousset @ncharles ?

**Comment by fanf** [comment]
This is outdated and not needed anymore

**Pull request closed without merging**